### PR TITLE
Initial commit for Oracle cloud infrastructure - Updated

### PR DESCRIPTION
**Pull request by nalsaber**
##### SUMMARY
Update license and logging as recommended by (Revert oracle #56364)

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
Oracle cloud infrastructure Manage Virtual Cloud Networks(VCN) module

##### ADDITIONAL INFORMATION
Files Submitted:
lib/ansible/module_utils/oracle/init.py
lib/ansible/module_utils/oracle/oci_logging.yaml
lib/ansible/module_utils/oracle/oci_utils.py
lib/ansible/modules/cloud/oracle/init.py
lib/ansible/modules/cloud/oracle/oci_vcn.py
lib/ansible/plugins/doc_fragments/oracle.py
lib/ansible/plugins/doc_fragments/oracle_creatable_resource.py
lib/ansible/plugins/doc_fragments/oracle_display_name_option.py
lib/ansible/plugins/doc_fragments/oracle_name_option.py
lib/ansible/plugins/doc_fragments/oracle_tags.py
lib/ansible/plugins/doc_fragments/oracle_wait_options.py

### CODE REVIEW

**Comment by nalsaber** [comment]
@mattclay please approve. I would like to backport this change to 2.8 also.

File: lib/ansible/modules/cloud/oracle/oci_vcn.py [hunk 1]
```diff
@@ -23,7 +20,7 @@
     - This module allows the user to create, delete and update virtual cloud networks(VCNs) in OCI.
       The complete Oracle Cloud Infrastructure Ansible Modules can be downloaded from
       U(https://github.com/oracle/oci-ansible-modules/releases).
-version_added: "2.8"
+version_added: "2.9"
```
Comment (mattclay) [inline lib/ansible/modules/cloud/oracle/oci_vcn.py]: Leave the version added as 2.8 -- we'll ignore that CI failure and merge (assuming everything else is OK).

Comment (nalsaber) [reply lib/ansible/modules/cloud/oracle/oci_vcn.py]: @mattclay done

**Comment by abadger** [comment]
The licensing changes look good to me.  At some point in the future you may need to look into using the module_utils/basic.py log handling instead of your ad hoc stuff but this is already an improvement over the current status so +1 from me.

**Comment by mattclay** [comment]
@nalsaber Thanks for the quick response on updating the license.

@abadger I've opened a backport PR for the license changes in stable-2.8: https://github.com/ansible/ansible/pull/56417

**Comment by nalsaber** [comment]
> The licensing changes look good to me. At some point in the future you may need to look into using the module_utils/basic.py log handling instead of your ad hoc stuff but this is already an improvement over the current status so +1 from me.
Sure, I will check it.

Thanks for the quick merge!

**Pull request merged**